### PR TITLE
preloader 

### PR DESCRIPTION
**Pull request by felizj17**
added a style sheet and the image in the public folder.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -39,5 +44,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script src = "./app.js"></script>
```
Comment (anktrj) [inline public/index.html]: This is not required

File: src/App.js [hunk 2]
```diff
@@ -2,7 +2,10 @@ import React from "react";
 import { Link, Route, BrowserRouter as Router, Switch } from "react-router-dom";
 import BinarySearch from "./components/binarySearch/BinarySearch";
 import LinearSearch from "./components/linearSearch/LinearSearch";
-
+ window.addEventListener('load', () =>{
+  const preload = document.querySelector('.preload')
+  preload.classList.add('preload-finish')
+ })
```
Comment (anktrj) [inline src/App.js]: This is also no required

File: public/index.html [hunk 3]
```diff
@@ -29,6 +30,10 @@
   <body style="margin: 0; background-color: #1b1b1b;">
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div class = "preload">
+      <img src="./visualrithmPreloader.gif" alt="robot processing">
+      <h2 id = "preloadText" >Please wait while we get things ready...</h2>
+    </div>
```
Comment (anktrj) [inline public/index.html]: move these lines inside <div id="root"> {Your code} </div>

**Review by ankiiitraj — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 3 above)
Comment (felizj17) [reply public/index.html]: sorry, I'm unsure where it should be placed

Comment (anktrj) [reply public/index.html]: Ok move all these lines inside the div with id `root`

**Review by ankiiitraj — APPROVED** [review]

**Comment by anktrj** [comment]
Thanks for contributing @felizj17

**Pull request merged**